### PR TITLE
Fix clear ug

### LINKED ISSUE #144: [PE-D][Tester D] Clear Confirmation Message does not appear correctly
To replicate: enter the clear command `clear`. When the confirmation box appears, click "Yes".

Expected: As stated in the User Guide, the output message should change to `All student data has been cleared.`

Actual:
Unfortunately, the message is not updated even after clicking "Yes" on the confirmation box.

-------------
Labels: `type.FunctionalityBug` `severity.Low`
original: lawruixi/ped#2

### LINKED ISSUE #149: [PE-D][Tester E] Delete feature: Error message mismatch
According to UG, expected output should be `Error! Email not found`

However I got this instead:

-------------
Labels: `severity.Low` `type.FeatureFlaw`
original: yanghengtang/ped#3

### LINKED ISSUE #161: [PE-D][Tester B] Unknown use of square brackets in UG
There are some use of square brackets which surrounds the value e.g. `t/[TUTORIAL]` rather than the entire tag e.g. `[t/TUTORIAL]`. This suggest that the value is optional but not the tag. However, this is not the observed behavior in the application.

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: McNaBry/ped#5

### PR DESCRIPTION
**Pull request by neyapraveen**
Fix messages in ug.
Closes #144 
Closes #161 
Closes #149

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ StudentConnect is a solution for hassle-free team formation for students to brow
 6. [FAQ](#faq)
 7. [Known issues](#known-issues)
 8. [Command summary](#command-summary)
+9. [Appendix](#appendix)
```
Comment (maj0-0) [inline docs/UserGuide.md:43]: thanks for adding appendix!

**Review by maj0-0 — COMMENTED** [review]
LGTM!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -120,14 +121,18 @@ Provides the requirements for forming a group.
...
+
 * GUI: All students' details are removed from student list.
-* Message: `Opened confirmation window. Please ensure you use the exit command when exiting StudentConnect for successful reset.`
-* Message (Once confirmed): `All student data has been cleared.`
   ![result for 'clear' pop-up](images/clearPopUp.png)
-  ![result for 'clear' GUI](images/clear.png)
+  ![result for 'clear' GUI](images/clearUI.png)
+
+#### Expected Output(Failure or Cancellation):
+If clear fails or you press `cancel` on the confirmation pop-up:
```
Comment (wnchan) [inline docs/UserGuide.md:133]: instead of clear, change to 'the clear command'

File: docs/UserGuide.md [hunk 3]
```diff
@@ -120,14 +121,18 @@ Provides the requirements for forming a group.
...
 * GUI: All students' details are removed from student list.
-* Message: `Opened confirmation window. Please ensure you use the exit command when exiting StudentConnect for successful reset.`
-* Message (Once confirmed): `All student data has been cleared.`
   ![result for 'clear' pop-up](images/clearPopUp.png)
-  ![result for 'clear' GUI](images/clear.png)
+  ![result for 'clear' GUI](images/clearUI.png)
+
+#### Expected Output(Failure or Cancellation):
+If clear fails or you press `cancel` on the confirmation pop-up: 
+* GUI: All students' details remain on student list.
```
Comment (wnchan) [inline docs/UserGuide.md:134]: 'the student list'

Comment (neyapraveen) [reply docs/UserGuide.md]: Done both!

**Pull request merged**